### PR TITLE
[product partnerships] Added AWS Inspector

### PR DESCRIPTION
**Pull request by rahul188**
# Summary

A concise description of the changes being introduced. Please review the pre-merge checklist section to validate this pull request is ready for review and merge. **If it is not ready, please mark the pull request as a draft.**

_The owners of this repo are not experts in the subject matter of the quickstarts. We review for the quickstart to be functional and for security risks. If you are seeking feedback on the content of the quickstart, please seek out a subject matter expert. If you are not an internal NR contributor, we can do our best to connect you with a content reviewer._

## Pre merge checklist

- [x] Did you check you NRQL syntax? - Does it work?
- [X] Did you include a Data source and Documentation reference?
- [X] Are all documentation links publicly accessible?
- [X] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [X] Did you check your descriptive content for spelling and grammar errors?
- [X] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [X] Does the PR contain a screenshot for each of your dashboards?
- [X] Do your screenshots show data?
- [X] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [X] Did you check that your alerts actually work?

### CODE REVIEW

**Review by mdumpati — CHANGES_REQUESTED** [review]
@rahul188 Can you please give the space between title, for your reference purpose please find this link https://docs.aws.amazon.com/inspector/v1/userguide/inspector_assessments.html

**Comment by rahul188** [comment]
@mdumpati, I have submitted the fix.

File: dashboards/amazon-inspector/amazon-inspector.json [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+            },
+            "visualization": {
+              "id": "viz.markdown"
+            },
+            "rawConfiguration": {
+              "text": "# Amazon Inspector\nAmazon Inspector is a vulnerability management service that continuously scans your AWS workloads for software vulnerabilities and unintended network exposure. Amazon Inspector automatically discovers and scans running Amazon EC2 instances, container images in Amazon Elastic Container Registry (Amazon ECR), and AWS Lambda functions for known software vulnerabilities and unintended network exposure."
+            }
+          },
+          {
+            "title": "Total findings by AssessmentTargetName",
```
Comment (mdumpati) [inline dashboards/amazon-inspector/amazon-inspector.json:77]:
```suggestion
            "title": "Total findings by assessment target name",
```

File: dashboards/amazon-inspector/amazon-inspector.json [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+              "platformOptions": {
+                "ignoreTimeRange": false
+              },
+              "yAxisLeft": {
+                "zero": true
+              }
+            }
+          },
+          {
+            "title": "Total findings by AssessmentTemplateName",
```
Comment (mdumpati) [inline dashboards/amazon-inspector/amazon-inspector.json:109]:
```suggestion
            "title": "Total findings by assessment template name",
```

File: dashboards/amazon-inspector/amazon-inspector.json [hunk 3]
```diff
@@ -0,0 +1,271 @@
...
+              "platformOptions": {
+                "ignoreTimeRange": false
+              },
+              "yAxisLeft": {
+                "zero": true
+              }
+            }
+          },
+          {
+            "title": "Total healthy agents by AssessmentTargetName",
```
Comment (mdumpati) [inline dashboards/amazon-inspector/amazon-inspector.json:141]:
```suggestion
            "title": "Total healthy agents by assessment target name",
```

File: dashboards/amazon-inspector/amazon-inspector.json [hunk 4]
```diff
@@ -0,0 +1,271 @@
...
+              "platformOptions": {
+                "ignoreTimeRange": false
+              },
+              "yAxisLeft": {
+                "zero": true
+              }
+            }
+          },
+          {
+            "title": "Total healthy agents by AssessmentTemplateName",
```
Comment (mdumpati) [inline dashboards/amazon-inspector/amazon-inspector.json:173]:
```suggestion
            "title": "Total healthy agents by assessment template name",
```

File: dashboards/amazon-inspector/amazon-inspector.json [hunk 5]
```diff
@@ -0,0 +1,271 @@
...
+              "platformOptions": {
+                "ignoreTimeRange": false
+              },
+              "yAxisLeft": {
+                "zero": true
+              }
+            }
+          },
+          {
+            "title": "Total matching agents by AssessmentTargetName",
```
Comment (mdumpati) [inline dashboards/amazon-inspector/amazon-inspector.json:205]:
```suggestion
            "title": "Total matching agents by assessment target name",
```

File: dashboards/amazon-inspector/amazon-inspector.json [hunk 6]
```diff
@@ -0,0 +1,271 @@
...
+              "platformOptions": {
+                "ignoreTimeRange": false
+              },
+              "yAxisLeft": {
+                "zero": true
+              }
+            }
+          },
+          {
+            "title": "Total matching agents by AssessmentTemplateName",
```
Comment (mdumpati) [inline dashboards/amazon-inspector/amazon-inspector.json:237]:
```suggestion
            "title": "Total matching agents by assessment template name",
```

File: dashboards/amazon-inspector/amazon-inspector.png
Comment (mdumpati) [inline dashboards/amazon-inspector/amazon-inspector.png:1]: @rahul188 After changing titles then replace the image also.

**Review by mdumpati — COMMENTED** [review]
@rahul188 Please check my comments

File: quickstarts/aws/amazon-inspector/config.yml [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+    url: >-
+      https://docs.newrelic.com/docs/infrastructure/amazon-integrations/aws-integrations-list/amazon-inspector-monitoring-integration/
+keywords:
+  - aws
+  - amazon web services
+  - inspector
+dashboards:
+  - amazon-inspector
+dataSourceIds:
+  - amazon-cloudwatch-metric-streams
```
Comment (pkosanam) [inline quickstarts/aws/amazon-inspector/config.yml:29]: alert-policies are added to the file changes. Could you please attach the alert-policy to this config.yml file.

File: dashboards/amazon-inspector/amazon-inspector.json (hunk 1 above)
Comment (aj-NR) [reply dashboards/amazon-inspector/amazon-inspector.json:77]: @mdumpati As discussed we will continue the same way. Please take ahead this PR for approval if there are no more comments

File: alert-policies/amazon-inspector/HighAssessmentRunFindings.yml [hunk 8]
```diff
@@ -0,0 +1,27 @@
+name: High Assessment Run Findings
+
+description: |+
+  This alert is triggered if the number of Assessment Run Findings exceeds 100 for 10 minutes.
```
Comment (pkosanam) [inline alert-policies/amazon-inspector/HighAssessmentRunFindings.yml:4]:
```suggestion
  This alert is triggered if the number of Assessment Run Findings exceeds 100 for 5 minutes.
```

File: alert-policies/amazon-inspector/HighAssessmentRunFindings.yml [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+
+# List of Critical and Warning thresholds for the condition
+terms:
+  - priority: CRITICAL
+    # Operator used to compare against the threshold.
+    operator: ABOVE
+    # Value that triggers a violation
+    threshold: 100
+    # Time in seconds; 120 - 3600
+    thresholdDuration: 600
```
Comment (pkosanam) [inline alert-policies/amazon-inspector/HighAssessmentRunFindings.yml:21]: Threshold Duration has to be changed if the text is changed to 5 minutes

File: alert-policies/amazon-inspector/HighAssessmentRunFindings.yml [hunk 10]
```diff
@@ -0,0 +1,27 @@
...
+    # How many data points must be in violation for the duration
+    thresholdOccurrences: ALL
+
```
Comment (pkosanam) [inline alert-policies/amazon-inspector/HighAssessmentRunFindings.yml:24]: Kindly add a warning condition too.

**Review by pkosanam — CHANGES_REQUESTED** [review]

File: quickstarts/aws/amazon-inspector/config.yml (hunk 7 above)
Comment (rahul188) [reply quickstarts/aws/amazon-inspector/config.yml]: I have added alertPolicies.

cc - @pkosanam @mdumpati

File: alert-policies/amazon-inspector/HighAssessmentRunFindings.yml (hunk 8 above)
Comment (rahul188) [reply alert-policies/amazon-inspector/HighAssessmentRunFindings.yml]: @pkosanam , addressed, I have changed the duration.

File: alert-policies/amazon-inspector/HighAssessmentRunFindings.yml (hunk 9 above)
Comment (rahul188) [reply alert-policies/amazon-inspector/HighAssessmentRunFindings.yml]: @pkosanam , addressed, I have changed the duration.

File: alert-policies/amazon-inspector/HighAssessmentRunFindings.yml (hunk 10 above)
Comment (rahul188) [reply alert-policies/amazon-inspector/HighAssessmentRunFindings.yml:24]: @pkosanam , addressed, I have added the warning condition.

File: dashboards/amazon-inspector/amazon-inspector.png
Comment (pkosanam) [reply dashboards/amazon-inspector/amazon-inspector.png:1]: as the above titles are not changed, resolving this comment.

**Review by pkosanam — APPROVED** [review]
LGTM

**Review by rahulbasu-nr — APPROVED** [review]
Looks good to me.

**Review by mdumpati — APPROVED** [review]
LGTM

**Pull request merged**